native/src/main/java/io/ballerina/lib/aws/redshift/Constants.java

```diff
@@ -39,8 +39,18 @@ public static final class ClientConfiguration {
         public static final BString CONNECTION_POOL_OPTIONS = StringUtils.fromString("connectionPool");
         public static final BString OPTIONS = StringUtils.fromString("options");
         public static final BString PROPERTIES = StringUtils.fromString("properties");
+        public static final BString SECURE_SOCKET = StringUtils.fromString("ssl");
     }
 
+    public static final class SSL {
+        public static final BString SSL_MODE = StringUtils.fromString("ssl");
+        public static final BString ROOT_CERT = StringUtils.fromString("rootcert");
+        public static final BString SSL_FACTORY_ARG = StringUtils.fromString("sslfactory");
+        public static final BString SSL_FACTORY_VALUE =
+                StringUtils.fromString("com.amazon.redshift.ssl.NonValidatingFactory");
```

Comment:
Isn't this `NonValidatingFactory` class related to creating SSL connections without validating the server's SSL certificate?